### PR TITLE
[NO JIRA] Add documentation in the README

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,41 @@ A React implementation of Echoes, Sonar's Design System.
...
-TODO - runs storybook, probably through:
+### VSCode Configuration
+
+We recommend VSCode to work on this project. There is a `.vscode` folder containing:
+
+- A list of recommended extensions
+  - Install them through the marketplace
+- A template for necessary (and recommended) settings: `settings.template.json`
+  - Copy it and save it as `settings.json`
+  - You must open the project directly. Adding its folder in an existing workspace might not work, as the typescript configuration muts be defined at workspace level.
```
Comment (gregaubert) [inline README.md:15]:
```suggestion
  - You must open the project directly. Adding its folder in an existing workspace might not work, as the typescript configuration must be defined at workspace level.
```

File: README.md [hunk 2]
```diff
@@ -4,28 +4,41 @@ A React implementation of Echoes, Sonar's Design System.
...
+
+### Run storybook
+
+Both these commands run the storybook server in `dev` mode:
```
Comment (gregaubert) [inline README.md:19]:
```suggestion
The following command runs both storybook server in `dev` mode and the vite build in watch mode:
```

File: README.md [hunk 3]
```diff
@@ -4,28 +4,41 @@ A React implementation of Echoes, Sonar's Design System.
...
 
 ```bash
 yarn dev
+yarn storybook
```
Comment (gregaubert) [inline README.md:23]: Only yarn dev is enough

File: package.json [hunk 4]
```diff
@@ -14,9 +14,9 @@
   },
   "packageManager": "yarn@4.0.2",
   "scripts": {
-    "dev": "echo 'coming soon'",
+    "dev": "vite build --watch && yarn storybook",
```
Comment (gregaubert) [inline package.json:17]:
```suggestion
    "dev": "vite build --watch & yarn storybook",
```

Only one `&` to run in parallel instead of in sequence

**Review by gregaubert — CHANGES_REQUESTED** [review]
Currently the `dev` script doesn't work since it's running the two commands in sequence

File: package.json (hunk 4 above)
Comment (jeremy-davis-sonarsource) [reply package.json]: I forgot to save the file :facepalm:

**Review by gregaubert — APPROVED** [review]

**Pull request merged**